gocourse05/zooobservation/animal/animal_test.go

```diff
@@ -0,0 +1,70 @@
+package animal
+
+import (
+	"testing"
+
+	"zooobservation/camera"
+	"zooobservation/common"
+)
+
+func TestAnimal_Move(t *testing.T) {
+	direct := common.Left
+	var historyItems []common.HistoryItem
+
+	tiger := Animal{
+		Id: 1,
+		Camera: &camera.DayCamera{
+			Screenshot: "day_screenshot.png",
+		},
+		Species: "tiger",
+	}
+
+	history, err := tiger.Move(direct, historyItems)
```

Comment:
Здається, так буде працювати:
```suggestion
	history, err := tiger.Move(direct, nil)
```